src/main/java/teammates/ui/output/FeedbackResponseLog.java

```diff
@@ -0,0 +1,26 @@
+package teammates.ui.output;
+
+import java.util.List;
+
+/**
+ * The response log of a single feedback session.
+ */
+public class FeedbackResponseLog {
```

Comment:
Perhaps `FeedbackSessionLog`? (I'm not really sure what response log means in this context!)